tests/integration/snapshots/integration__update__update_all_dependencies-2.snap

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Why does this file exist? I assume it is left around from previous tests but no longer used.

Additionally I don't think these files should exist:
- `update_named_dependency-2`
- `update_one_dependency`
- `updated_named_dependency-2`
- `updated_named_dependency`

Reply:
You'd be correct, they're from old tests that I forgot to clean up. They're cleaned up and there shouldn't be any leftovers now.